src/vue-testing-library.js

```diff
@@ -73,7 +73,26 @@ function render(
     unmount: () => wrapper.unmount(),
     html: () => wrapper.html(),
     emitted: () => wrapper.emitted(),
-    setProps: props => wrapper.setProps(props),
+    rerender: ({props}) => {
+      wrapper.unmount()
+      mountedWrappers.delete(wrapper)
+
+      wrapper = mount(
+        TestComponent,
+        merge({
+          attachTo: container,
+          global: {
+            plugins,
+          },
+          ...mountOptions,
+          props,
+          // ...additionalOptions,
+        }),
+      )
```

Comment:
Would it make sense to create come sort of a `mountComponent` function (or something with a better name 😇) to avoid repeating this code block? Both rendering methods must have the same keys, and having them in a single place looks like a good thing.

Reply:
Hm, it could even include the unwrap + add lines below 🤔

Reply:
[React Testing Library](https://github.com/testing-library/react-testing-library/blob/master/src/pure.js#L54) has something like that. At a first glance I think it should be doable.

Reply:
I added my first iteration of the function. Let me know what you think! The 2 original blocks of code were slightly different due to the overwrite of `props`, so I had to include that in the arrow function. But besides that, I think it's okay. If you're fine with it, you can resolve this piece. If not, you can let me know! :smile: